src/interface/src/app/services/map.service.ts

```diff
@@ -36,7 +37,19 @@ export class MapService {
   );
   readonly conditionNameToDisplayNameMap$ = new BehaviorSubject<Map<string, string>>(new Map<string, string>());
 
-  constructor(private http: HttpClient) {
+  readonly selectedRegion$ = new BehaviorSubject<Region | null>(null);
+
+  private readonly destroy$ = new Subject<void>();
+
+  constructor(private http: HttpClient, private sessionService: SessionService) {
+
+    this.sessionService
```

Comment:
Are you getting the Angular change detected error in your dev console with this set?